exercicios/para-casa/entregas/Geice Sousa/resolucaoGeice.js

```diff
@@ -0,0 +1,78 @@
+class Filme {
+  constructor(titulo, categoriaAluguel){
+    this.titulo = titulo,
+    this.categoriaAluguel = categoriaAluguel
+  }
+
+  set alterarCategoriaAluguel(novaCategoria){
+    this.categoriaAluguel = novaCategoria
+  }
+}
+
+class Cliente{
+  constructor(nome){
+    this.nome = nome,
+    this.historicoFilmesDeAlugados = [],
+    this.pontosFidelidade = 0
+  }
+
+}
+
```

Comment:
As dependências são injetadas corretamente nas instâncias de Locacao, mas a referência ao cliente na função adicionaLocacao não está claro, pra mim. eu te dou a sugestão de deixar dessa forma:  
class Locacao {
  constructor(cliente) {
    this.filmesAlugados = [];
    this.cliente = cliente;
  }

  adicionarFilme(filme) {
    this.filmesAlugados.push(filme);
  }